packages/ra-ui-materialui/src/input/ArrayInput/ArrayInput.spec.tsx

```diff
@@ -17,36 +17,13 @@ import { ArrayInput } from './ArrayInput';
 import { SimpleFormIterator } from './SimpleFormIterator';
 import { useFormContext } from 'react-hook-form';
 import { GlobalValidation, ValidationInFormTab } from './ArrayInput.stories';
+import { useArrayInput } from './useArrayInput';
 
 describe('<ArrayInput />', () => {
-    it('should pass its record props to its child', async () => {
```

Comment:
That's a problem that it doesn't work. Without a RecordContext, we can't use a field as child of ArrayInput.

Reply:
ArrayInput does not set the RecordContext, SimpleFormIteratorItem does